src/actions.js

```diff
@@ -36,11 +36,13 @@ module.exports = {
 						id: 'gpi',
 						default: '1',
 						regex: Regex.NUMBER,
+						useVariables: true,
 					},
 				],
 				callback: async (event) => {
```

Comment:
You should do this to get context:
```suggestion
				callback: async (event, context) => {
```